downloader_library/build.gradle

```diff
@@ -15,8 +15,14 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.txt'
         }
     }
+lintOptions {
```

Comment:
If this is disabled, why is this suppressed in some files?

Reply:
Disabling in this file doesn't suppress the error flag of longlog tag while the code is analyzed in android studio. I forgot to remove the disable line in build.gradle file.

Reply:
I will remove it now.